### PR TITLE
enable  create session from an endpoint

### PR DESCRIPTION
**Pull request by acezen**
enable  create session from an endpoint

### CODE REVIEW

**Comment by acezen** [comment]
cc/ @sighingnow @lidongze0629

**Review by sighingnow — APPROVED** [review]
LGTM.

**Review by siyuan0322 — APPROVED** [review]
LGTM

File: coordinator/gscoordinator/coordinator.py [hunk 1]
```diff
@@ -146,10 +146,11 @@ def __init__(self, launcher, dangling_seconds, log_level="INFO"):
 
         # dangling check
         self._dangling_seconds = dangling_seconds
-        self._dangling_detecting_timer = threading.Timer(
-            interval=self._dangling_seconds, function=self._cleanup, args=(True,)
-        )
-        self._dangling_detecting_timer.start()
+        if self._dangling_seconds != -1:
```
Comment (lidongze0629) [inline coordinator/gscoordinator/coordinator.py:149]: Would it be better?
```python
if self._dangling_seconds >= 0:
    xxxx
```

File: python/graphscope/client/session.py [hunk 2]
```diff
@@ -273,7 +273,7 @@ def __init__(
         # update other optional params
         self._config_params.update(kw)
 
-        self._config_params["addr"] = None
+        self._config_params["addr"] = kw.pop("addr", None)
```
Comment (lidongze0629) [inline python/graphscope/client/session.py:276]: Maybe you can update the session doc if we support param `addr`.

**Review by lidongze0629 — APPROVED** [review]

File: coordinator/gscoordinator/coordinator.py (hunk 1 above)
Comment (acezen) [reply coordinator/gscoordinator/coordinator.py]: good suggestion, fixed.

**Pull request merged**